gql/queries/EventDetail.gql

```diff
@@ -1,58 +1,137 @@
-# Replace libcalId with slug later
-#import "~/gql/fragments/Image"
-query EventDetail($slug: [String!]) {
-  entry(section: "event", slug: $slug) {
-    ... on event_event_Entry {
-      id
-      typehandle
-      sectionHandle
-      title
-      eventLocation
-      image: heroImage {
-        ... on heroImage_heroImage_BlockType {
-          image {
-            ...Image
-          }
-        }
-      }
-      to: uri
-      slug
-      libcalId
-      publicUrl
-      date {
-        ... on date_date_BlockType {
-          id
-          startTime
-          endTime
-        }
-      }
-      eventType {
-        ... on eventType_Category {
-          id
-          title
-        }
-      }
-      eventDescription
-      associatedLocations {
-        ... on location_location_Entry {
-          id
-          to: uri
-          title
-          summary
-        }
-      }
-      requiresRegistration
-      onlineProvider: libcalEventOnlineProvider
-      onlineJoinURL: onlineEventUrl
-      eventTags {
-        ... on eventTags_Tag {
-          id
-          title
-        }
-      }
-      presenter
-      moreInformation
-      eventOwner
-    }
-  }
-}
+#import "~/gql/fragments/Image"
+#import "~/gql/fragments/collections/DefaultFpbWithCardsAndHighlight.gql"
+query EventDetail($slug: [String!]) {
+    entry(section: "event", slug: $slug) {
+        ... on event_event_Entry {
+            id
+            typehandle
+            sectionHandle
+            title
+            eventLocation
+            image: heroImage {
+                ... on heroImage_heroImage_BlockType {
+                    image {
+                        ...Image
+                    }
+                }
+            }
+            to: uri
+            slug
+            libcalId
+            publicUrl
+            date {
+                ... on date_date_BlockType {
+                    id
+                    startTime
```

Comment:
I think we need to account for the timezone. Japanese 100A Advanced Modern Japanese event shows up as 12pm-4pm on the front end, but the entry says 9am-1pm.

You should be able to use the formatDateTime directive: 
`startTime @formatDateTime(format: "h:i a", timezone: "America/Los_Angeles")`

Reply:
Made a ticket to address this in both our components and queries